oeis.py

```diff
@@ -413,6 +413,23 @@ def A007953(start: int = 0, limit: int = 20) -> Collection[int]:
     return sequence
 
 
+@oeis
+def A000120(start: int = 0, limit: int = 20) -> Collection[int]:
+    """1's-counting sequence: number of 1's in binary
+    expansion of n (or the binary weight of n).
+    """
+    sequence = []
+
+    for n in range(start, start + limit):
+        count = 0
+        binary = bin(n)
+        binary = binary[2:].replace("0", "")
+        count = binary.count("1")
```

Comment:
Why are you couting `1`s on a line having only ones? I mean, those are identicals:
```pycon
>>> "{:b}".format(42).replace("0", "").count("1")
3
>>> len("{:b}".format(42).replace("0", ""))
3
>>> "{:b}".format(42).count("1")
3
```

I find the shorter one more readable.